content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -0,0 +1,180 @@
+top of the manufacturing or inventory order after the check fails. Clicking :guilabel:`Quality
+Alert` opens a quality alert form on a new page. For a complete guide on how to fill out the quality
+alert form, view the documentation on :ref:`quality alerts
+<quality/quality_management/quality-alerts>`.
+
+On a work order
+---------------
+
+When configuring a |QCP| that is triggered during manufacturing, a specific work order can also be
+specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, a *Measure* quality check is created for that specific work order, rather than the
+manufacturing order as a whole.
+
+*Measure* quality checks created for work orders must be processed from the tablet view. To do so,
+begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`.
+Select a manufacturing order that includes a work order for which a quality check is required. Open
+the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and then clicking
+the :guilabel:`📱 (tablet)` button on the order's line.
+
+With tablet view open, complete the steps listed on the left side of the screen until the *Measure*
+quality check step is reached. Upon reaching the check, the instructions for how to take the
+measurement appear at the top of the screen. Enter the measured value in the :guilabel:`Measure`
+field above the instructions, the click :guilabel:`Validate`.
+
+If the measurement entered is within the range specified in the :guilabel:`Tolerance` section of the
+|QCP|, the quality check passes and the tablet view moves on to the next step of the work order.
+However, if the measurement entered is outside of the specified range, a pop-up window appears,
+titled :guilabel:`Quality Check Failed`.
+
+The body of the :guilabel:`Quality Check Failed` pop-up window shows a warning message that states,
+"You measured # units and it should be between # units and # units.", as well as the instructions
+entered in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the pop-up, two
+buttons appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
+
+.. image:: measure_check/measure-check-failed.png
+   :align: center
+   :alt: The "Quality Check Failed" pop-up window.
+
+If the measurement is incorrect and should be changed, select :guilabel:`Correct Measure`. Doing so
+opens a new pop-up window, titled :guilabel:`Quality Check`. Enter the corrected measurement in the
+:guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check and move on the
+next step of the work order.
+
+If the measurement is correct, click :guilabel:`Confirm Measure` instead, and the quality check
+fails. Follow any instructions that were listed on the :guilabel:`Quality Check Failed` pop-up
+window.
+
+If a quality alert must be created, do so by clicking the :guilabel:`☰ (menu)` button, and selecting
```

Comment:
when we use emojis, we describe what the emoji looks like in case the person's browser doesn't load the emoji. 
```suggestion
If a quality alert must be created, do so by clicking the :guilabel:`☰ (three horizontal lines)` menu button,
```
or
```suggestion
If a quality alert must be created, do so by clicking the :guilabel:`☰ (hamburger)` button,
```